extensions/job_management/standard/requirements/job-management/update/REQ_response.adoc

```diff
@@ -0,0 +1,7 @@
+[[req_job-management_update_response]]
+[requirement]
+====
+[%metadata]
+label:: /req/job-management/update/response
+part:: A successful execution of the operation SHALL be reported as a response with a HTTP status code `200` or `204`.
```

Comment:
We solely use 204 in openEO. What is 200 used for / why two response codes?

Reply:
A server could decide to return the updated contents directly, such that a client does not need to do another `GET /jobs/{jobId}` to obtain it. This is left up to the discretion of the server, since the client does not necessarily need to do `GET /jobs/{jobId}` either (it can guess which field was updated and updating its definition on its side).